symbolicationWebService.py

```diff
@@ -207,22 +224,16 @@ def ReadConfigFile():
       return False
 
   # Check for section names
-  if not set(["General", "Log"]).issubset(set(configParser.sections())):
-    logging.error("'General' and 'Log' sections are mandatory in the config file")
+  if not set(["General", "Log", "DiskCache"]).issubset(set(configParser.sections())):
+    logging.error("'General', 'Log' and 'DiskCache' sections are mandatory in the config file")
```

Comment:
i don't know if we want a disk cache for local debugging. the user might change code, regenerate their local .sym files but forget to restart the Snappy server. that would result in an invalid disk-cache state